resources/test/rpc_schema_hashing_V1.json

```diff
@@ -2245,7 +2245,7 @@
                 "properties": {
                   "Withdraw": {
                     "items": {
-                      "$ref": "#/components/schemas/UnbondingPurse"
+                      "$ref": "#/components/schemas/WithdrawPurse"
```

Comment:
I am not sure how people use these schemas and whether their pipeline is scripted enough to recognize that the path for the type's schema has changed.

/cc @Fraser999